examples/worldview/src/features/kepler/hooks.js

```diff
@@ -71,47 +75,37 @@ export const useKeplerKeyframes = keplerLayers => {
     // console.log(keyframes, keplerLayers);
     if (filterKeyframe) {
       // TODO: Support more than one filter.
-      keyframes.hubble_timeFilter = new FilterValueKeyframes(filterKeyframe);
+      keyframes.kepler_timeFilter = new FilterValueKeyframes(filterKeyframe);
     }
     return keyframes;
   }, [filterKeyframe, layerKeyframe, keplerLayers]);
 
   return getKeplerKeyframes;
 };
 
-export const usePrepareKeplerFrame = keplerLayers => {
+export const useKeplerFrame = (keplerLayers = []) => {
   const dispatch = useDispatch();
+  const frame = useSelector(frameSelector);
```

Comment:
Now we can use a selector for the whole frame instead of having to call `getFrame` on every single keyframe object in user code.